src/liblsquic/lsquic_frame_reader.c

```diff
@@ -698,11 +700,12 @@ code_str_to_reason (const char *code_str, int code_len)
     };
 
     long code;
-    char code_buf[ code_len + 1 ];
+    char * code_buf = malloc(code_len + 1 );
```

Comment:
I think the thing to realize here is that `code_str_to_reason()` is only ever called with code_len = 3.  We can avoid the malloc.